addon/gql/queries/search-form-question.graphql

```diff
@@ -1,11 +1,11 @@
 #import "ember-caluma/gql/fragments/question-info"
 
-query SearchFormQuestion($slug: String!, $search: String) {
+query SearchFormQuestion($slug: String!, $search: String, $archived: Boolean) {
   allForms(slug: $slug) {
     edges {
       node {
         slug
-        questions(search: $search, isArchived: false) {
+        questions(search: $search, isArchived: $archived) {
```

Comment:
This here seems to be the culprit. If I change this back to a static `false` the refresh of the question list works as expected.